packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -9,64 +9,119 @@
         class="bg-white rounded-b max-h-72vh pb-90px overflow-scroll"
       >
         <div class="h-full">
-          <div class="p-16px">
+          <div
+            class="p-16px"
+            data-cy="release-highlights"
+          >
             <h1 class="font-medium mt-4px text-center mb-32px tracking-tighter text-22px text-gray-1000">
               {{ t('majorVersionWelcome.title') }}
             </h1>
             <div class="mb-16px">
               <ExternalLink
-                href="https://on.cypress.io/changelog#11-0-0"
+                href="https://on.cypress.io/changelog#12-0-0"
                 class="font-bold text-indigo-500"
               >
-                11.0.0
+                12.0.0
               </ExternalLink>
               <span class="font-light pl-10px text-gray-500 text-14px">
-                Released {{ versionReleaseDates['11'] }}
+                Released {{ versionReleaseDates['12'] }}
               </span>
             </div>
             <div class="children:mb-16px">
+              <p>
+                For a complete list of updates and breaking changes in v12.0.0, please review our
+                <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
+                <ExternalLink href="https://on.cypress.io/changelog#12-0-0">
+                  changelog
+                </ExternalLink>.
+              </p>
+
               <h2 class="font-bold text-18px text-jade-1000">
-                Component Testing Released
+                Testing Multi-Origin WorkFlows
```

Comment:
```suggestion
                Testing Multi-Origin Workflows
```

Capital F looked odd to me?